lib/pure/os.nim

```diff
@@ -103,9 +106,24 @@ proc normalizePathEnd(path: var string, trailingSep = false) =
     path = $DirSep
 
 proc normalizePathEnd(path: string, trailingSep = false): string =
+  ## A version of ``normalizePathEnd()``
+  ## See also:
+  ## * `normalizePathEnd proc <#normalizePathEnd,string,bool>`_ for the
+  ## in-place version.
+  runnableExamples:
+    when defined(posix):
+      assert normalizePathEnd("/lib/", trailingSep = true) == "/lib/"
+      assert normalizePathEnd("", trailingSep = true) == "/"
+      assert normalizePathEnd("/lib", trailingSep = false) == "/lib"
+      assert normalizePathEnd("lib//", trailingSep = false) == "lib"
+      assert normalizePathEnd("lib", trailingSep = false) == "lib"
+      assert normalizePathEnd("/lib//", trailingSep = true) == "/lib/"
   result = path
   result.normalizePathEnd(trailingSep)
 
+when (NimMajor, NimMinor) >= (1, 1):
```

Comment:
This is not what `since` template is for?. :thinking:

Reply:
The problem is that these procs already existed; we're just exporting them as of 1.1, but not before.

Reply:
That doesnt matter (implementation detail), what matters is public api

Reply:
so... how about using `since` ?